src/commcare_cloud/ansible/roles/keepalived/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+# Variables for role keepalived
+keepalived_auth_pass: "1111"
+keepalived_role: "MASTER"
+keepalived_router_id: "52"
+keepalived_shared_iface: "eth0"
+keepalived_shared_ip: "192.168.1.1"
+monit_process: "nginx"
+keepalived_priority: "200"
+keepalived_backup_priority: "100"
+script_user: "root"
+notify_script_path: "/etc/keepalived/notify_script"
+smtp_server: "smtp.gmail.com"
```

Comment:
These should probably be blank in the role defaults and then set per environment in `public.yml`